arrows/serialize/json/tests/test_serialize.cxx

```diff
@@ -194,6 +339,8 @@ TEST( serialize, detected_object )
     {
       EXPECT_EQ( *(o_it->first), *(d_it->first) );
       EXPECT_EQ( o_it->second, d_it->second );
+      o_it++;
```

Comment:
Considering these are tests and are less subject to performance requirements, you can disregard this comment with regards to this PR, but in general, it's best to pre-increment when possible. Post-increment requires an extra copy which can get costly when the iterator objects are large or when there are a lot of them.

Reply:
Good to know! This was easy enough to change in this case: a7e446f368e2708cefd14cdd367d795516350dda